src/v/resource_mgmt/storage.h

```diff
@@ -35,6 +35,7 @@ class api;
 class node;
 
 class eviction_policy {
+public:
```

Comment:
Perhaps this addition is in the wrong commit?

Reply:
popped the `public:` out into the prev commit to avoid the weirdness